use std::error::Error;

pub fn gen_algolia_product_records() -> Result<(), Box<dyn Error>> {


    Ok(())
}